test/e2e/overridepolicy_test.go

```diff
@@ -425,6 +428,197 @@ var _ = ginkgo.Describe("[OverridePolicy] apply overriders testing", func() {
 				})
 		})
 	})
+
+	ginkgo.Context("[FieldOverrider] apply field overrider testing to update YAML values in ConfigMap", func() {
+		var configMapNamespace, configMapName string
+		var configMap *corev1.ConfigMap
+
+		ginkgo.BeforeEach(func() {
+			configMapNamespace = testNamespace
+			configMapName = configMapNamePrefix + rand.String(RandomStrLength)
+			propagationPolicyNamespace = testNamespace
+			propagationPolicyName = configMapName
+			overridePolicyNamespace = testNamespace
+			overridePolicyName = configMapName
+
+			configMapData := map[string]string{
+				"nginx.yaml": `
+server:
+  listen: 80
+  server_name: localhost
+  location /:
+    root: /usr/share/nginx/html
+    index: 
+      - index.html
+      - index.htm
+  error_page:
+    - code: 500
+    - code: 502
+    - code: 503
+    - code: 504
+  location /50x.html:
+    root: /usr/share/nginx/html
+`,
+			}
+			configMap = helper.NewConfigMap(configMapNamespace, configMapName, configMapData)
+			propagationPolicy = helper.NewPropagationPolicy(propagationPolicyNamespace, propagationPolicyName, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: configMap.APIVersion,
+					Kind:       configMap.Kind,
+					Name:       configMap.Name,
+				},
+			}, policyv1alpha1.Placement{
+				ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+					ClusterNames: framework.ClusterNames(),
+				},
+			})
+
+			overridePolicy = helper.NewOverridePolicy(overridePolicyNamespace, overridePolicyName, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: configMap.APIVersion,
+					Kind:       configMap.Kind,
+					Name:       configMap.Name,
+				},
+			}, policyv1alpha1.ClusterAffinity{
+				ClusterNames: framework.ClusterNames(),
+			}, policyv1alpha1.Overriders{
+				FieldOverrider: []policyv1alpha1.FieldOverrider{
+					{
+						FieldPath: "/data/nginx.yaml",
+						YAML: []policyv1alpha1.YAMLPatchOperation{
+							{
+								SubPath:  "/server/location/root",
```

Comment:
This is likely incorrect.

```
server:
  listen: 80
  server_name: localhost
  location /: # Note the / here
```

Reply:
you should use `"/server/location ~1/root"`
See https://jsonpatch.com/#json-pointer for more details.